File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L5/04/Kasus3.ts
import readline from 'readline';

function hitungTinggi(feet: number, inches: number): number {
    return feet * 12 + inches;
}

function hitungBmi(weight: number, height: number): number {
    return weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254));
}

function cekBmi(bmi: number): void {
    if (bmi < 18.5) {
        console.log("Underweight");
    } else if (bmi < 25) {
        console.log("Normal");
    } else if (bmi < 30) {
        console.log("Overweight");
    } else {
        console.log("Obese");
    }
}

async function main(): Promise<void> {
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });

    const weightPrompt = await new Promise<number>((resolve) => {
        rl.question("Enter weight in pounds: ", (weight) => {
            resolve(parseFloat(weight));
        });
    });

    const feetPrompt = await new Promise<number>((resolve) => {
        rl.question("Enter feet: ", (feet) => {
            resolve(parseFloat(feet));
        });
    });

    const inchesPrompt = await new Promise<number>((resolve) => {
        rl.question("Enter inches: ", (inches) => {
            resolve(parseFloat(inches));
        });
    });

    rl.close();

    const bmi = hitungBmi(weightPrompt, hitungTinggi(feetPrompt, inchesPrompt));
    console.log("BMI is " + bmi);
    cekBmi(bmi);
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L6/04/Kasus3.py
import math

def hitung_tinggi(feet, inches):
    return feet * 12 + inches

def hitung_bmi(weight, feet, inches):
    return weight * 0.45359237 / ((hitung_tinggi(feet, inches) * 0.0254) * (hitung_tinggi(feet, inches) * 0.0254))

def cek_bmi(bmi):
    if bmi < 18.5:
        return "Underweight"
    elif bmi < 25:
        return "Normal"
    elif bmi < 30:
        return "Overweight"
    else:
        return "Obese"

if __name__ == "__main__":
    weight = float(input("Enter weight in pounds: "))
    feet = float(input("Enter feet: "))
    inches = float(input("Enter inches: "))
    
    bmi = hitung_bmi(weight, feet, inches)
    print(f"BMI is {bmi:.2f}")
    print(cek_bmi(bmi))

